Dockerfile

```diff
@@ -5,9 +5,9 @@ FROM node:16-alpine
 WORKDIR /app
 
 # Copy package.jsons first to install
-COPY package.json /app/package.json
-COPY frontend/package.json /app/frontend/package.json
-COPY server/package.json /app/server/package.json
+COPY package.json package-lock.json ./
```

Comment:
This is equivalent, but it might fit better with the other COPY commands to move it into `/app/` rather than `./`. Or we can change the others to `./frontend`, for example.